browsermobproxy/client.py

```diff
@@ -269,17 +269,20 @@ def timeouts(self, options):
                          params)
         return r.status_code
 
-    def remap_hosts(self, address, ip_address):
+    def remap_hosts(self, hostmap):
```

Comment:
This is a breaking change. I would rather have it so that we have the original arguments for at least another release, with the new one as a keyword argument and then we can move it over.